corehq/apps/hqadmin/utils.py

```diff
@@ -8,9 +8,11 @@
 
 def pillow_seq_store():
     for pillow in get_all_pillow_instances():
-        checkpoint = pillow.get_checkpoint()
-        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint['_id'])
-        if not created and force_seq_int(checkpoint['seq']) < force_seq_int(store.seq) - EPSILON:
+        checkpoint = pillow.checkpoint
+        store, created = PillowCheckpointSeqStore.objects.get_or_create(checkpoint_id=checkpoint.checkpoint_id)
+        db_seq = checkpoint.get_current_sequence_id()
+        store_seq = force_seq_int(store.seq) or 0
```

Comment:
this was a separate issue that failed on the comparison if nothing was in the store.